### ISSUE TITLE
Looking for experienced admin / developer of IOMAD, to help figure out how to operate and develop for iomad

### ISSUE THREAD

**Issue by virtuman** [iomad/iomad#1896]
hello, i've been trying to figure out limits of IOMAD and best practices for development custom functionalities and plugins, but struggling with lack of documentation and was thinking to take on the task of how fresh iomad contributor moves from newbie to operating iomad in production.

I would love to find a person or a team, that has some knowledge in the following:
- (tech) maintaining repo for locally developed plugins and IOMAD origin repo, so can continuously pull from main origin, and propose pull requests if any core changes necessary (could or should it be used as git submodule, or best practices for iomad?)
- (tech) same as above, but for docker image (ie. i feel like iomad repo should build Docker image, that we can use as base for plugin development (if it's supposed to be supported this way, for example our development should go into /local folder in form of plugins, and could be easily COPY during build over existing IOMAD base image)
- (tech) what's better to run iomad : php-fpm+nginx or apache, or facebook's c-php engine?
- (tech) Running in docker for local dev
- (tech) Running in kubernetes for prod env
- (admin) configure iomad for multi-tenancy with proper file permissions, and other specifics
- (admin) add company signup page (or is there such function out of the box)
- (admin) teach what proper permissions are needed to operate multi-tenant site, so that new company gets all needed access, but not too much, so they don't mess up entire instance of iomad, as well as their own
- (admin) setup catalog-like pages, or should it be offloaded to woocommerce-like integrations

basically i would like to gather all this info, document in wiki of this repo or i'll create a separate iomad documentation site, and take charge of maintaining it as well as various Troubleshooting guides, admin setup sections, development onboarding, etc.

while i'm looking for someone who can work with me to aggregate all the info, i would sincerely appreciate any relevant replies re some of the questions above, as well as all other pains that you've faced when starting to work with iomad

thank you for any help with getting one more fellow developer onboarded into the community.
